netbox/extras/forms/model_forms.py

```diff
@@ -374,3 +378,99 @@ class Meta:
             'assigned_object_type': forms.HiddenInput,
             'assigned_object_id': forms.HiddenInput,
         }
+
+
+EMPTY_VALUES = ('', None, [], ())
+
+
+class FormMetaclass(forms.models.ModelFormMetaclass):
```

Comment:
We should rename this to avoid confusion with the stock class.

```suggestion
class ConfigRevisionFormMetaclass(forms.models.ModelFormMetaclass):
```